README.md

```diff
@@ -2,130 +2,131 @@
-> 
-> - If an issue affects only a single component, it should be raised in that
->   components repository.
+## Introduction
 
-## Kata Containers repositories
+Kata Containers is an open source project and community working to build a
+standard implementation of lightweight Virtual Machines (VMs) that feel and
+perform like containers, but provide the workload isolation and security
+advantages of VMs.
 
-### CI
+## Getting started
 
-The [CI](https://github.com/kata-containers/ci) repository stores the Continuous
-Integration (CI) system configuration information.
+See the [installation documentation](docs/install).
 
-### Community
+## Documentation
 
-The [Community](https://github.com/kata-containers/community) repository is
-the first place to go if you want to use or contribute to the project.
+See the [official documentation](docs)
+(including installation guides, developer guide, design document and
+how-to's).
 
-### Code Repositories
+## Community
 
-#### Kata Containers-developed components
+To learn more about the project, its community and governance, see the
+[community repository](https://github.com/kata-containers/community). This is
+the first place to go if you wish to contribute to the project.
 
-##### Agent
+## Getting help
 
-The [`kata-agent`](src/agent/README.md) runs inside the
-virtual machine and sets up the container environment.
+See the [community](#community) section for ways to contact us.
 
-##### KSM throttler
+### Raising issues
 
-The [`kata-ksm-throttler`](https://github.com/kata-containers/ksm-throttler)
-is an optional utility that monitors containers and deduplicates memory to
-maximize container density on a host.
+#### Current version
 
-##### Runtime
+Please raise an issue
+[in this repository](https://github.com/kata-containers/kata-containers/issues).
 
-The [`kata-runtime`](src/runtime/README.md) is usually
-invoked by a container manager and provides high-level verbs to manage
-containers.
+#### Kata Containers 1.x versions
 
-##### Trace forwarder
+Please raise an issue in the [Kata Containers 1.x component repository](#kata-containers-1x-components)
+that seems most appropriate.
 
-The [`kata-trace-forwarder`](src/trace-forwarder) is a component only used
-when tracing the [agent](#agent) process.
+If in doubt, raise an issue
+[in the Kata Containers 1.x runtime repository](https://github.com/kata-containers/runtime/issues).
 
-#### Additional
+## Developers
 
-##### Kernel
+### Components
 
-The hypervisor uses a [Linux\* kernel](https://github.com/kata-containers/linux) to boot the guest image.
+| Component | Type | Description |
+|-|-|-|
+| [agent-ctl](tools/agent-ctl) | utility | Tool that provides low-level access for testing the agent. |
+| [agent](src/agent) | core | Management process running inside the virtual machine / POD that sets up the container environment. |
+| [osbuilder](tools/osbuilder) | infrastructure | Tool to create "mini O/S" rootfs and initrd images for the hypervisor. |
```

Comment:
Instead of having a column for "core", "utility", etc, I believe it might be clearer to make separate tables, and have "Core components", "Utilities", etc. wdyt?

Reply:
I just noticed we lost the ksm throttler. I assume this was intentional. Can someone point me to the discussion for that decision. so that I understand the rationale?

Reply:
> Can someone point me to the discussion for that decision. so that I understand the rationale?

There isn't really one I'm afraid :) However, I've raised #1073 so there is some more visibility of this.

Reply:
Regarding the columns, I haven't done that as that would result in more tables and also single row tables for documentation (unless we want to classify docs as "core" or it isn't really as I had in mind "core" meaning "essential code component" which docs are not ;)